spec/creditsafe/client_spec.rb

```diff
@@ -1,13 +1,40 @@
+# frozen_string_literal: true
 require 'spec_helper'
 require 'creditsafe/client'
+require 'timecop'
 
 URL = 'https://webservices.creditsafe.com/GlobalData/1.3/'\
-      'MainServiceBasic.svc'.freeze
+      'MainServiceBasic.svc'
 
 RSpec.describe(Creditsafe::Client) do
+  notifications = []
   let(:username) { "b" }
   let(:password) { "c" }
+  before(:all) do
+    ActiveSupport::Notifications.subscribe do |*args|
+      notifications << ActiveSupport::Notifications::Event.new(*args)
+    end
+  end
+  before(:each) { notifications = [] }
```

Comment:
Unfortunately the execution context in `subscribe` doesn't let me put these in an `@notifications` variable or anything like that. (The rest is the canonical `ActiveSupport::Notifications` collect-all-events recipe).